qiskit/transpiler/passes/layout/sabre_layout.py

```diff
@@ -253,11 +253,11 @@ def run(self, dag):
         self.property_set["layout"] = Layout(initial_layout_dict)
         # If skip_routing is set then return the layout in the property set
         # and throwaway the extra work we did to compute the swap map.
-        # We also skip routing here if the input circuit is split over multiple
-        # connected components and there is a shared clbit between any
-        # components. We can only reliably route the full dag if there is any
-        # shared classical data.
-        if self.skip_routing or shared_clbits:
```

Comment:
Do we still need to compute `shared_clbits`? It doesn't seem used anymore :)

Reply:
Yeah, we can drop it since it's not used anymore

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/10000/commits/07679e005679a823a49d16463c1042e4ea91b06a